u_parrots.go

```diff
@@ -938,3 +941,263 @@ func removeRC4Ciphers(s []uint16) []uint16 {
 	}
 	return s[:sliceLen]
 }
+
+// FingerprintClientHello returns a ClientHelloSpec which is based on the ClientHello that is passed in as the data argument
+// If the ClientHello passed in has extensions that are not recognized or cannot be handled, it will return a non-nil error and a nil `*ClientHelloSpec` value
+// The `data` should be the ClientHello record as produced by `crypto/tls.clientHelloMsg.marshal()`
+// ie. it should not contain the full tls record, just the handshake message as outlined in https://tools.ietf.org/html/rfc5246#section-7.4
```

Comment:
The backticks won't format in Godoc unfortunately.  Nor will the newlines as-is.  You need a blank line between the lines, which will create a new paragraph.  This is how the documentation currently renders:

![image](https://user-images.githubusercontent.com/4043597/89844480-ed0e2480-db38-11ea-83e5-b167291318c2.png)

[This is a kind of nifty primer on Godoc formatting](https://pkg.go.dev/github.com/natefinch/godocgo?tab=doc) if you're looking for more info.

Reply:
:shakes-fist-at-sky:!!!! Thanks for the tips!!!